### ISSUE TITLE
[BUG]: Playwright getCompliance label with outputFilenameTimestamp true causes error

### ISSUE THREAD

**Issue by EvilKarter** [IBMa/equal-access#1755]
### Project

accessibility-checker for Node

### Browser

Chrome

### Operating system

Windows

### Description

Hi I encountered a problem when I used playwright and the node accessibility-checker. 

When I run my playwright script with getCompliance label with the configuration below (outputFilenameTimestamp:true) I always get the following error on the console: 
```shell
Error: ENOENT: no such file or directory, open 'my\path\results\summary_2023-11-24T08:43:41.325Z.json'
    at Object.open (node:internal/fs/sync:78:18)
    at Object.openSync (node:fs:565:17)
    at writeFileSync (node:fs:2288:35)
    at MyFS.writeFileSync (my\path\node_modules\accessibility-checker\lib\ACHelper.js:88:32)
    at Function.<anonymous> (my\path\node_modules\accessibility-checker\lib\common\report\ReporterManager.js:354:48)
    at step (my\path\node_modules\accessibility-checker\lib\common\report\ReporterManager.js:59:23)
    at Object.next (my\path\node_modules\accessibility-checker\lib\common\report\ReporterManager.js:40:53)
    at fulfilled (my\path\node_modules\accessibility-checker\lib\common\report\ReporterManager.js:31:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'my\\path\\results\\summary_2023-11-24T08:43:41.325Z.json'
}
``` 

It doesn't matter what I call the label. 
As soon as I set outputFilenameTimestamp to true I get the error that it could not find the file summary+timestamp. 

I also get the result file in the directory. But not with the name from the error, but with the name from the label.

Is this a bug or have I missed an obvious connection?

my used versions: 
    "accessibility-checker": "^3.1.64",
    "playwright": "^1.40.0"
### Steps to reproduce

my test script
```javascript
const { chromium }  = await import('playwright');
const aChecker = await import('accessibility-checker');

(async () => {
  const browser = await chromium.launch();
  const context = await browser.newContext();
  const page = await context.newPage();

  await page.goto('https://github.com/IBMa/equal-access');
  // Adjust the configuration here

  // Check accessibility
  const report = await aChecker.getCompliance(page, 'myLabel');

  await browser.close();
})();
``` 
my .achecker.yml
```
ruleArchive: versioned
policies:
    - IBM_Accessibility
    - WCAG_2_2
failLevels:
    - violation
    - potentialviolation
reportLevels:
    - violation
    - potentialviolation
outputFormat: json
outputFolder: results
outputFilenameTimestamp: true
``` 

Call test script
```shell
node test.js
```

**Comment by tombrunet** [comment]
I'm guessing a Windows issues with the :'s in the filename. We'll look.

**Comment by ErickRenteria** [comment]
Merged
